foo_uie_console/main.cpp

```diff
@@ -506,6 +545,9 @@ LRESULT ConsoleWindow::on_hook(HWND wnd, UINT msg, WPARAM wp, LPARAM lp)
 
             menu.append_submenu(L"&Edge style", edge_style_submenu.detach());
 
+            menu.append_separator();
```

Comment:
I think this separator isn't needed personally (would also be consistent with the caption context menu that `ConsoleWindow::get_menu_items()` is for).